### PR TITLE
Add a chapter to the XR manual that improves the startup script.

### PR DESCRIPTION
**Pull request by BastiaanOlij**
This PR adds a new chapter to the XR manual detailing the contents of our enhanced startup script that we used in our XR demos in the Godot demos repository.

Our introduction contains a bare minimum script that forms the basis for all XR interfaces. This script adds specific details for OpenXR only and adds further context to a number of signals emitted by the OpenXR interface.
It also has further details on the unique nature of refresh rates on XR headsets and how to properly set things up.

### CODE REVIEW

File: tutorials/xr/a_better_xr_start_script.rst [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+-----------------
+
+This signal is emitted by OpenXR when our game gets focus. This is done at the completion of our startup, but it can also be emitted when the user exits a system menu, or put their headset back on.
+
+Note also that when your game starts while the user is not wearing their headset, the game stays in 'visible' state until the user puts their headset on.
+
+.. warning::
+
+  It is thus important to keep your game paused while in visible mode. If you don't the game will keep on running while your user isn't interacting with your game.
+  Also when the game returns to focussed mode, suddenly all controller and hand tracking is reenabled and could have game breaking consequences if you do not react to this accordingly.
```
Comment (skyace65) [inline tutorials/xr/a_better_xr_start_script.rst:335]:
```suggestion
  Also when the game returns to focussed mode, suddenly all controller and hand tracking is re-enabled and could have game breaking consequences if you do not react to this accordingly.
```

I haven't done a full read through of everything yet, but this error is why the build is failing.

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: tutorials/xr/a_better_xr_start_script.rst (hunk 1 above)
Comment (BastiaanOlij) [reply tutorials/xr/a_better_xr_start_script.rst]: Cheers, I went to bed straight after submitting so hadn't checked the CI

**Review by skyace65 — CHANGES_REQUESTED** [review]
Finally read through everything, the instructions and explanations made sense. Not seeing any grammar errors.

There's a ton of lines that need to be broken up into multiple lines though:
93
95
171
173
175
267
269
275
380
415
448

**Comment by BastiaanOlij** [comment]
> There's a ton of lines that need to be broken up into multiple lines though:

Don't things just wrap automatically? Happy to start every sentence on a new line but generally I stick to paragraphing of information that belongs together. Formatting is as much about conveying information as the actual text.

**Comment by skyace65** [comment]
They do but it makes it a bit harder to edit when everything is on one very long line, we don't have those anywhere else in the docs. I'm not sure I understand your point about paragraphing, When I say split into multiple lines I mean like lines 6 and 7 on this page, that's still one paragraph. I'm not asking you to break up those lines and add blank space between them, sorry if that wasn't clear.

**Comment by BastiaanOlij** [comment]
> They do but it makes it a bit harder to edit when everything is on one very long line, we don't have those anywhere else in the docs. I'm not sure I understand your point about paragraphing, When I say split into multiple lines I mean like lines 6 and 7 on this page, that's still one paragraph. I'm not asking you to break up those lines and add blank space between them, sorry if that wasn't clear.

Ah, so it's not adding in a hard newline and break the flow.. Ok no problem, I can clean that up

**Comment by BastiaanOlij** [comment]
@skyace65  Let me know if it's better, the painful thing of making these changes is that all the line numbers change with it, so it was hard to check if I fixed all the places you identified :)

**Review by skyace65 — APPROVED** [review]
Looks good now!

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**